apps/iir_blur/iir_blur_generator.cpp

```diff
@@ -63,33 +63,41 @@ Func blur_cols_transpose(Func input, Expr height, Expr alpha, bool skip_schedule
             // Really for an IIR on the GPU you should use a more
             // specialized DSL like RecFilter, but we can schedule it
             // in Halide adequately, we just can't extract any
-            // parallelism from the scan dimension.
-
-            // We also have no way to store kernel intermediates on
-            // the GPU in global memory, and we can't fuse together
-            // multiple update stages into a single loop, so we
-            // currently have to launch a whole lotta
-            // kernels. store_with would ameliorate this to some
-            // degree, or making store_in get you to global memory on
-            // the GPU. All in all, this is a pretty mediocre schedule
-            // for an IIR blur.
-
-            // 3.39ms on a 2060 RTX
+            // parallelism from the scan dimension. Most GPUs will be
+            // heavily under-utilized with this schedule and thus
+            // unable to hide the memory latencies to L2.
 
+            // 2.06ms on a 2060 RTX
             Var xi, yi;
-            blur.compute_root();
+            transpose.compute_root()
+                .tile(x, y, xi, yi, 32, 32)
```

Comment:
magic numbers like 32 are probably worth pulling into named constants and explaining why, at least in intended-as-example code like this

Reply:
Done. It's the cuda warp size.